<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tablaPacientes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Nombre </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Riesgo</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="botonRegistrar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar Paciente</string>
    </property>
   </widget>
   <widget class="QFrame" name="PanelRiesgo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>150</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Detalles del paciente </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BPM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBPM">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>spO2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSpO2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperatura:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemp">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="botonSimular">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Simular signos </string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonExportar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exportar Datos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonGrafico">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver Gr√°fico</string>
    </property>
   </widget>
   <zorder>tablaPacientes</zorder>
   <zorder>PanelRiesgo</zorder>
   <zorder>botonSimular</zorder>
   <zorder>botonRegistrar</zorder>
   <zorder>botonExportar</zorder>
   <zorder>botonGrafico</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
